torchtune/modules/loss/loss_protocols.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Protocol
+
+import torch
+
+
+class SFTLossWithProjection(Protocol):
```

Comment:
Probably needs to be ``SFTLossWithOutputProj`` or something. Projection is too vague.

Reply:
I agree that this name is confusing. I think we should just standardize on "fused" or "linear", or "chunked". All the names have issues which we've discussed but if we're consistent at least people should be able to learn the term quickly.